### PR TITLE
Updated documentation for charts (cloud)

### PR DESCRIPTION
**Pull request by kickoke**
I updated the topic on charts in the cloud, mainly refining the copy. I also renamed the file from interact-new-charts.md into interact-charts-cloud.md to differentiate between the Agent documentation, which is called interact-charts.mdx
I also updated the links.

### CODE REVIEW

File: docs/dashboard/interact-charts.mdx [hunk 1]
```diff
@@ -133,7 +133,7 @@ your needs.
 
 - Dashboard
   - [How the dashboard works](/docs/dashboard/how-dashboard-works)
-  - [Netdata Cloud · Interact with new charts](/docs/cloud/visualize/interact-new-charts)
```
Comment (hugovalente-pm) [inline docs/dashboard/interact-charts.mdx:136]: For this specific change we need to do it on the Agent repo, please check [here](https://github.com/netdata/learn#ingest-and-process-documentation-files).

If it is files under /docs/cloud/ and the sidebar that refers to this section this is the right repo, for the others we need to start the update from the Agent repo (netdata/netdata)

**Review by hugovalente-pm — CHANGES_REQUESTED** [review]
If you just change the stuff on /docs/cloud this is good to go, with the dependency on the agent it would mean we would need to _hold_ the fix on the `new` on the filename until we have the stuff from the Agent merged.

Suggestion just change the `/docs/cloud/visualize/interact-new-charts.md` stuff and on a next PR the filename could be changed

File: docs/dashboard/interact-charts.mdx (hunk 1 above)
Comment (kickoke) [reply docs/dashboard/interact-charts.mdx:136]: Changed the filename back to interact-new-charts
I might need to update/ restructure the docs soon, so the naming concern isn't urgent.

**Comment by kickoke** [comment]
Changed back the filename to interact-new-charts as a quick fix. However, the documentation probably needs restructuring in the future.

**Review by hugovalente-pm — APPROVED** [review]

**Pull request merged**